dom.go

```diff
@@ -346,25 +348,76 @@ func RenderBody(body Component) {
 	}
 	doRestore(nil, body, nil, nextRender)
 	// TODO: doRestore skip == true here probably implies a user code bug
-	doc := js.Global.Get("document")
+	doc := global.Get("document")
 	if doc.Get("readyState").String() == "loading" {
 		doc.Call("addEventListener", "DOMContentLoaded", func() { // avoid duplicate body
-			doc.Set("body", nextRender.Node)
+			doc.Set("body", nextRender.node)
 		})
 		return
 	}
-	doc.Set("body", nextRender.Node)
+	doc.Set("body", nextRender.node)
 }
 
 // SetTitle sets the title of the document.
 func SetTitle(title string) {
-	js.Global.Get("document").Set("title", title)
+	global.Get("document").Set("title", title)
 }
 
 // AddStylesheet adds an external stylesheet to the document.
 func AddStylesheet(url string) {
-	link := js.Global.Get("document").Call("createElement", "link")
+	link := global.Get("document").Call("createElement", "link")
 	link.Set("rel", "stylesheet")
 	link.Set("href", url)
-	js.Global.Get("document").Get("head").Call("appendChild", link)
+	global.Get("document").Get("head").Call("appendChild", link)
+}
+
+var global jsObject = wrapObject(js.Global)
+
+type jsObject interface {
+	Set(key string, value interface{})
+	Get(key string) jsObject
+	Call(name string, args ...interface{}) jsObject
+	String() string
+	Bool() bool
+}
+
+func wrapObject(j *js.Object) jsObject {
+	if j == nil {
+		return nil
+	}
+	if j == js.Undefined {
+		panic("TODO")
```

Comment:
Can we not just return `js.Undefined`?  I tried rebasing my PR against master, but this breaks things pretty badly.